packages/api-explorer/src/ApiExplorer.tsx

```diff
@@ -116,6 +127,26 @@ export const ApiExplorer: FC<ApiExplorerProps> = ({
     return () => unregisterEnvAdaptor()
   }, [])
 
+  useEffect(() => {
+    // reconcile local storage state with URL or vice versa
+    if (initialized) {
+      const sdkLanguage = searchParams.get('sdk')
+      if (sdkLanguage && getLanguageByAlias(sdkLanguage)) {
```

Comment:
This check means that if a user passes in `sdk=invalid` we are defaulting to what they previously had in the store. Alternatively we could just push them to `all`. @jkaster  any preference?

Reply:
I think an invalid language should clear any configured preference and reset to the default behavior